lib/danger/ci_source/azure_pipelines.rb

```diff
@@ -15,15 +16,35 @@ module Danger
   #
   # ### Token Setup
   #
-  # Add the `DANGER_GITHUB_API_TOKEN` to your environment variables.
+  # #### GitHub
+  #
+  # You need to add the `DANGER_GITHUB_API_TOKEN` environment variable, to do this, go to your build definition's variables tab.
+  #  #
+  # #### Azure Git
+  #
+  # You need to add the `DANGER_VSTS_API_TOKEN` and `DANGER_VSTS_HOST` environment variable, to do this,
+  # go to your build definition's variables tab. The `DANGER_VSTS_API_TOKEN` is your vsts personal access token.
+  # Instructions for creating a personal access token can be found [here](https://www.visualstudio.com/en-us/docs/setup-admin/team-services/use-personal-access-tokens-to-authenticate).
+  # For the `DANGER_VSTS_HOST` variable the suggested value is `$(System.TeamFoundationCollectionUri)$(System.TeamProject)`
+  # which will automatically get your vsts domain and your project name needed for the vsts api.
   #
   class AzurePipelines < CI
+    class << self
+      def tfsgit_slug(env)
+        project_name = env["SYSTEM_TEAMPROJECT"]
+        repo_name = env["BUILD_REPOSITORY_NAME"]
+
+        "#{project_name}/_git/#{repo_name}"
```

Comment:
@sphanley was kind of a surprised that this was all we needed in the slug.

I thought we the slug + PR # were the pieces of date we need to interact with the AzDo PR APIs.
I thought this would have been made up of organization as well?

Reply:
perhaps SYSTEM_TEAMPROJECT has the org in there?

Reply:
Per their current implementation, the org gets concatenated onto the `DANGER_VSTS_HOST` url, so we don't need to provide it as part of the slug. So all we need are project and repo, then PR number gets tacked on later.

Reply:
For example, in our pipeline we're setting the following variable: 
> DANGER_VSTS_HOST: "$(System.CollectionUri)$(System.TeamProject)"

Where System.TeamProject is MN.

Reply:
You can see [here](https://github.com/Progressive-Insurance/danger/pull/1/files#diff-8ca4efcb438d6d9f1abd76a8cf7ecb639660945ca2ce5b1fd6b6e7d8109a313bR150-R153) that we split it up into `project` (e.g. MN), _git (which is discarded), and `slug` (e.g. PGR-iOS). Then, [in vsts_api.rb](https://github.com/Progressive-Insurance/danger/blob/feature/merge-vsts-into-azure-pipelines/lib/danger/request_sources/vsts_api.rb#L25), you can see that only the `slug` variable containing the repo name is actually used in the URI. 

I was thinking to change it so that the `project` from the slug is actually used, but that would break current users since they have their `DANGER_VSTS_HOST` including the project name. Come to think of it, though, it might be possible to literally make the slug just be the repo name without even the `{project}/_git` prefix, since the project is being discarded.

Reply:
Yeah, reworked a few things and cleaned up the unnecessary discarded `project` parameter from the VSTSAPI class. Now the slug is even more puzzlingly simple!